src/Files.Uwp/DataModels/NavigationControlItems/DriveItem.cs

```diff
@@ -247,6 +238,13 @@ public async Task LoadDriveIcon()
             }
             Icon = await IconData.ToBitmapAsync();
         }
+
+        private void SizeChanged(object sender, PropertyChangedEventArgs e)
```

Comment:
(Additional note, you should adhere to the original name - in this case `PropertyChanged`)

But please remove that method as per my first comment